yt/frontends/stream/data_structures.py

```diff
@@ -431,7 +431,9 @@ def update_data(self, data):
 
 
 class StreamParticleFile(ParticleFile):
-    pass
+    def open_handle(self):
+        # stream is in-memory, so do not need to implement open_handle
+        pass
```

Comment:
IMO `pass` looks more like missing code in this context, when the method actually _is_ implemented by `return None`,so might as well make it explicit rather than implicit ? This isn't the only instance where I wanted to make this comment, but I'll avoid duplicating it.
```suggestion
        return None
```

Reply:
updated in the latest commit